coreax/__init__.py

```diff
@@ -59,8 +59,7 @@
     NystromApproximateKernel,
```

Comment:
The __init__ doc is out of date talking about "See :class:`@coreax.kernel.Kernel`" for flattening methods.

Reply:
Are you happy for me to simply remove everything after the first paragraph, seeing as everything else is now pretty much redudant?

Reply:
Yeah I think that is reasonable